src/libs/components/list/listDragComponent.jsx

```diff
@@ -15,10 +15,10 @@ import ListDragItem from "./listDragItem";
 class ListDragComponent extends Component {
   render() {
     const {
-      items, selectedItem, onSelect, onMove, onDrop, className,
+      items, selectedItem, onSelect, onMove, onDrop, ...otherProps
     } = this.props;
     return (
-      <List className={className}>
+      <List {...otherProps} >
```

Comment:
I am not a huge fan of spreading props like this because you will likely spread invalid props.

Reply:
List will validate them. I checked React docs and that the way they do.